src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,78 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.22.0.dev1 (10/04/2019)
+------------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Add black support as a v2 rule (#8350)
+  `PR #8350 <https://github.com/pantsbuild/pants/pull/8350>`_
+
+* Follow up on critical path algorithm (#8381)
+  `PR #8381 <https://github.com/pantsbuild/pants/pull/8381>`_
+
+* Use the newest pyflakes for internal Python linter (#8359)
+  `PR #8359 <https://github.com/pantsbuild/pants/pull/8359>`_
```

Comment:
Think this only impacts us internally (if I understand correctly how the contrib checker works). If so, should go under tooling.

Reply:
This will affect anyone using that contrib module.